game_eggs/steamcmd_servers/project_zomboid/egg-project-zomboid.json

```diff
@@ -28,44 +33,125 @@
-            "user_viewable": 1,
-            "user_editable": 1,
+            "user_viewable": true,
+            "user_editable": true,
             "rules": "required|string|max:64"
         },
         {
             "name": "Admin Username",
             "description": "Username for the admin account",
             "env_variable": "ADMIN_USER",
             "default_value": "admin",
-            "user_viewable": 1,
-            "user_editable": 1,
+            "user_viewable": true,
+            "user_editable": true,
             "rules": "required|string|max:20"
         },
         {
             "name": "Admin Password",
             "description": "Password for the admin account",
             "env_variable": "ADMIN_PASSWORD",
             "default_value": "password",
-            "user_viewable": 1,
-            "user_editable": 1,
+            "user_viewable": true,
+            "user_editable": true,
             "rules": "required|string|max:32"
         },
         {
             "name": "SteamPort",
             "description": "Sets the SteamPort1 option",
             "env_variable": "STEAM_PORT",
             "default_value": "8766",
-            "user_viewable": 1,
-            "user_editable": 0,
+            "user_viewable": true,
+            "user_editable": false,
             "rules": "required|string|max:20"
         },
         {
             "name": "Max Players",
             "description": "Maximum players to allow",
             "env_variable": "MAX_PLAYERS",
             "default_value": "10",
-            "user_viewable": 1,
-            "user_editable": 0,
+            "user_viewable": true,
+            "user_editable": false,
+            "rules": "required|string|max:20"
+        },
+        {
+            "name": "Steam Username",
+            "description": "Steam user setup. If a user has 2fa enabled it will most likely fail due to timeout. Leave blank for anon install.",
+            "env_variable": "STEAM_USER",
+            "default_value": "",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "nullable|string"
+        },
+        {
+            "name": "Steam Password",
+            "description": "Steam user setup. If a user has 2fa enabled it will most likely fail due to timeout. Leave blank for anon install.",
+            "env_variable": "STEAM_PASS",
+            "default_value": "",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "nullable|string"
+        },
+        {
+            "name": "Steam Auth Code",
+            "description": "Steam user setup. If a user has 2fa enabled it will most likely fail due to timeout. Leave blank for anon install.",
+            "env_variable": "STEAM_AUTH",
+            "default_value": "",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "nullable|string"
+        },
+        {
+            "name": "WINDOWS_INSTALL",
```

Comment:
Please don't add variables that are not necessary. Having many variables makes it messy, and this one, for example, will break the whole installation if someone sets it to 1.

zomboid doesn't need steam user or branch password to download it. Thus, all the steam variables are not required.